bazel/repository_locations.bzl

```diff
@@ -621,11 +621,11 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Rules for using foreign build systems in Bazel",
         project_desc = "Rules for using foreign build systems in Bazel",
         project_url = "https://github.com/bazelbuild/rules_foreign_cc",
-        version = "d54c78ab86b40770ee19f0949db9d74a831ab9f0",
-        sha256 = "e7446144277c9578141821fc91c55a61df7ae01bda890902f7286f5fd2f6ae46",
+        version = "4520018bc27dbd78b84d839cbe27a614ca3931f0",
+        sha256 = "7befe6d4e799a5cd0dc27cab1e96d41d018cc7e480adae3f45662c64f52d25f7",
         strip_prefix = "rules_foreign_cc-{version}",
         urls = ["https://github.com/bazelbuild/rules_foreign_cc/archive/{version}.tar.gz"],
-        release_date = "2020-10-26",
+        release_date = "2021-02-24",
```

Comment:
This version `rules_foreign_cc` makes a change to [fetch prebuild cmake and ninja](https://github.com/bazelbuild/rules_foreign_cc/commit/689c96aaa7337eb129235e5388f4ebc88fa14e87). I have some reservations about this as it tries to pull in dependencies that we have not defined. The way I was able to revert to prior behavior and have the compile work was to add  `rules_foreign_cc_dependencies(register_default_tools = False)` to `bazel/dependency_imports.bzl`. This might help your errors in CI. The deps look OK otherwise.

Reply:
@jsharpe New update based on the latest commit https://github.com/bazelbuild/rules_foreign_cc/commit/03cd811f50b16ce8ae07da3ef934f90fc1bfbd33

I now need `rules_foreign_cc_dependencies(register_default_tools = False, register_built_tools = False)` to prevent the import and build of `make`, `cmake`, `ninja` in `bazel/dependency_imports.bzl`

Reply:
```python
    rules_foreign_cc = dict(
        project_name = "Rules for using foreign build systems in Bazel",
        project_desc = "Rules for using foreign build systems in Bazel",
        project_url = "https://github.com/bazelbuild/rules_foreign_cc",
        version = "03cd811f50b16ce8ae07da3ef934f90fc1bfbd33",
        sha256 = "3ef6fab675b2b9bddecaed2f5169c3e80d2bc8d90a94ef0dc10f84b2b53056da",
        release_date = "2021-03-02",
        strip_prefix = "rules_foreign_cc-{version}",
        urls = ["https://github.com/bazelbuild/rules_foreign_cc/archive/{version}.tar.gz"],
        use_category = ["build"],
    ),
```